src/DevBetterWeb.Web/Pages/Admin/Books/BookViewModel.cs

```diff
@@ -19,6 +19,7 @@ public class BookViewModel
 	[Display(Name = "Book Category")]
 	public int? BookCategoryId { get; set; }
 
+	[StringLength(2000, ErrorMessage = "Details cannot be longer than 2000 characters.")]
```

Comment:
Might be able to use a constant, like MAX_BOOK_DESCRIPTION_LENGTH, so you don't have 2000 in quite so many places.